app/views/admin/worldwide_offices/_legacy_form.html.erb

```diff
@@ -0,0 +1,56 @@
+<%= form_for [:admin, worldwide_organisation,worldwide_office], html: {class: "well"} do |office_form| %>
+  <%= office_form.errors %>
+  <fieldset class="contact">
+    <%= office_form.fields_for(:contact) do |contact_form| %>
+      <%= contact_form.text_field :title %>
+      <%= contact_form.text_area :comments, rows: 2, placeholder: "Optional instructions for this contact e.g. opening hours, rules of use…" %>
+      <div class="form-group">
+        <%= office_form.label :worldwide_office_type_id, 'Office type' %>
+        <%= office_form.select :worldwide_office_type_id,
+            grouped_options_for_select(
+              legacy_worldwide_office_type_options,
+              office_form.object.worldwide_office_type_id || WorldwideOfficeType::Other.id
+            ),
+            {},
+            class: ['chzn-select', 'form-control'],
+            data: { placeholder: "Choose office type…" } %>
+      </div>
+      <%= render partial: 'admin/contacts/embed_details_for_form', locals: { contact: office_form.object.contact } %>
+      <%= render partial: 'admin/contacts/legacy_contact_type_details_form', locals: { contact_form: contact_form } %>
+      <div class="radio">
+        <p class="bold add-label-margin">Feature on home page?</p>
+        <% if worldwide_organisation.is_main_office?(office_form.object) %>
+          <p>Yes (as the main office)</p>
+        <% else %>
+          <%= office_form.labelled_radio_button('yes', :show_on_home_page, '1', checked: worldwide_organisation.office_shown_on_home_page?(office_form.object)) %>
+          <%= office_form.labelled_radio_button('no', :show_on_home_page, '0', checked: !worldwide_organisation.office_shown_on_home_page?(office_form.object)) %>
+        <% end %>
+      </div>
+
+      <fieldset class="services">
+        <legend>Services</legend>
+        <% # without this hidden field, once a service has been selected and saved, all services cannot be de-selected.
+           # http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-check_box-label-Gotcha %>
+        <%= hidden_field_tag 'worldwide_office[service_ids][]', nil %>
```

Comment:
These lines are supposed to be removed?

Reply:
Same as above, though also I wouldn't worry about anything in the legacy file since it will all be removed.

Reply:
ok